### PR TITLE
[Behat][Shop] Allow to go back in checkout

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Related tickets |
| License         | MIT

- [x] go back to the store from addressing step
- [x] go back to the addressing step from shipping step (by button and step label)
- [x] go back to the shipping step from payment step (by button and step label)
- [x] go back to the addressing step from payment step
- [x] go back to the payment step from order summary page (by button and step label)
- [x] go back to the shipping step from order summary page
- [x] go back to the addressing step form order summary page

### CODE REVIEW

File: src/Sylius/Behat/Context/Ui/CheckoutContext.php [hunk 1]
```diff
@@ -288,6 +305,44 @@ public function iCompleteTheShippingStep()
...
+     */
+    public function iDecideToChangeMyShippingMethod()
+    {
+        $this->paymentPage->changeShippingMethod();
+    }
+
+    /**
+     * @When I go to the addressing step
+     */
+    public function iGoToTheAddressingStep()
```
Comment (pamil) [inline src/Sylius/Behat/Context/Ui/CheckoutContext.php]: Shouldn't it throw an exception if we're not on shipping or payment page?

**Comment by pjedrzejewski** [comment]
Thanks Mateusz, please apply @pamil's fix in separate PR. :)

**Comment by Zales0123** [comment]
@pamil's comment is already applied, but not shown as outdated (it's not the line that has been changed ;))

File: src/Sylius/Bundle/ShopBundle/Resources/config/routing/checkout.yml [hunk 2]
```diff
@@ -49,6 +66,23 @@ sylius_shop_checkout_shipping:
...
+            repository:
+                method: find
+                arguments: [expr:service('sylius.cart_provider').getCart()]
+            state_machine:
+                graph: sylius_order_checkout
+                transition: reselect_shipping
+            redirect:
+                route: sylius_shop_checkout_shipping
+                parameters: []
+            permission: false
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/ShopBundle/Resources/config/routing/checkout.yml]: No need for that anymore.

**Pull request merged**